lib/scripts/postinstall.ts

```diff
@@ -0,0 +1,38 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as process from 'process';
+import * as util from 'util';
+
+declare const __dirname: string;
+
+const access = util.promisify(fs.access);
+const copyFile = util.promisify(fs.copyFile);
+const mkdir = util.promisify(fs.mkdir);
+
+async function notExists(p: string) {
+    try {
+        await access(p);
+        return false;
+    } catch (err) {
+        return true;
+    }
+}
+
+async function main() {
+    if (await notExists('./src/')) {
+        await mkdir('./src/');
+    }
+
+    if (await notExists('./src/styles/')) {
+        await mkdir('./src/styles/');
+    }
+
+    if (await notExists('./src/styles/_colors.scss')) {
+        await copyFile(path.resolve(__dirname, '../../src/styles/_colors.scss'), './src/styles/_colors.scss');

```

Comment:
is ../../src/styles/_colors.scss the path to the file in baseline

Reply:
no, it's the file in the current package in node_modules folder.